README.md

```diff
@@ -23,6 +23,11 @@ This app is created by the volunteers of [Public Knowledge Workshop](https://www
 - `yarn install`
 - `yarn start`
 
+### testing the project:
+- running the tests - `yarn playwright test`
```

Comment:
![image](https://github.com/hasadna/open-bus-map-search/assets/76536506/c5c1bb0f-e955-4ab8-b797-56facfaaf704)

we have script for it 
`npm run test:playwright`

In addition, What do you think about changing the scripts?
`
"scripts": {
  ...
  "test": "playwright test",
  "test:ui": "playwright test --ui"
  ...
}
`

Do we need the "react-scripts test" when we use playwright? 
if so we should add,
`"test:react-scripts": "react-scripts test"`
because we don't use it so often

Afterwards, we will change the docs accordingly.

Reply:
Yes, you right , my mistake. I meant `yarn run test:playwright` in the current scripts.
If you would like first to change the scripts and then change the documentation, the commands will be `yarn run test` / `yarn run test:ui`
Tell me what you prefer.

Reply:
having `yarn run test` and `yarn run test:ui` sounds great